src/platform/gui.h

```diff
@@ -171,7 +171,8 @@ class Menu {
     virtual ~Menu() {}
 
     virtual std::shared_ptr<MenuItem> AddItem(
-        const std::string &label, std::function<void()> onTrigger = std::function<void()>()) = 0;
+        const std::string &label, bool mnemonics = true,
```

Comment:
I think unless you put the `mnemonics` argument after `onTrigger` the default value for it is quite pointless.

Reply:
Thanks!